py/.craft.yml

```
@@ -25,6 +25,6 @@ targets:
 
 requireNames:
   - /^sentry_relay-.*-py2.py3-none-macosx_10_15_x86_64.whl$/
-  - /^sentry_relay-.*-py2.py3-none-manylinux2010_i686.whl$/
-  - /^sentry_relay-.*-py2.py3-none-manylinux2010_x86_64.whl$/
+  - /^sentry_relay-.*-py2.py3-none-.*manylinux2010_i686.whl$/
```

Would suggest to relax the regex here to allow `.*` after 2010 again, just in case this changes again.

```suggestion
  - /^sentry_relay-.*-py2.py3-none-.*manylinux2010_i686.*\.whl$/
```